source/hierarchy.txt

```diff
@@ -12,22 +12,210 @@ Organization, Project, and Deployment Hierarchy
    :depth: 1
    :class: onecol
 
-Intro statement
+Organizations, projects, and {+clusters+} are the building blocks
+of your {+service+} enterprise estate. At the organization level, you
+can implement security controls and create users that work across one or more projects. Projects offer a more fine-grained security isolation
+and authorization boundary, and {+clusters+} are your cloud databases
+in {+service+}.
+
+Use the foundational guidance on this page to plan the hierarchy and size for your organizations, projects, and {+clusters+}. This guidance optimizes security and performance from the start while aligning with
+your enterprise's billing and access needs.
 
 {+service+} Features and Best Practices for Hierarchy
 -----------------------------------------------------
 
-Content here
+You can use the following levels of hierarchy to define
+security settings and governance for your {+service+} enterprise estate:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - {+service+} Hierarchy Level
+     - Description
+
+   * - (Optional) Paying Organization
+     - One organization can be a paying organization for other
+       organization(s). The MongoDB account team must enable the paying organization when you establish the |service| subscription. A paying organization lets you set up :atlas:`cross-organization billing <billing/#std-label-cross-org-billing>` to share a
+       billing subscription across multiple organizations.
+
+   * - Organization
+     - Invoicing aggregates and generates at this level. Line items go
+       to cluster level and below. An organization often maps to a
+       business unit (BU) or application.
+
+   * - Project
+     - Security configuration for the data plane (including database
+       clusters and other data services) occurs at this level,
+       including network security. A project often maps to an application and environment (for example: Customer Portal app -
+       Production environment). For each project, there is one dedicated VPC created in GCP, and one VPC/VNet per region in AWS
+       and Azure. 
+
+   * - {+Cluster+}
+     - |service| provisions each {+cluster+} within the dedicated 
+       VPC/VNet(s) for a project. The security configuration is shared
+       among {+clusters+} in the project, except for user roles and authorization, which you can apply to actions on a {+cluster+},
+       database, and collection level.
+
+**DIAGRAM**
+
+Suggested Org and Project Hierarchies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _project-hierarchy-1:
+
+Suggested Hierarchy 1: Fewer Atlas Organizations
+````````````````````````````````````````````````
+
+This hierarchy, which creates fewer |service| organizations, might be useful if you have common teams and permissions across the business unit, and less than the (raiseable) limit of 250 projects per organization. 
```

Comment:
Suggest adding some sentence that clarifies that this is an example. Left an idea below.
Also question, if the limit is raisable, is it actually a limit?

```suggestion
Consider the following hierarchy for an example enterprise. This hierarchy, which creates fewer |service| organizations, might be useful if you have common teams and permissions across the business unit, and less than the raisable limit of 250 projects per organization. 
```